src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.models.Circle;
+import core.basesyntax.models.IsoscelesTrapezoid;
+import core.basesyntax.models.Rectangle;
+import core.basesyntax.models.RightTriangle;
+import core.basesyntax.models.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final String DEFAULT_COLOR = "rgb(255, 255, 255)";
```

Comment:
Consider replacing the RGB values in `DEFAULT_COLOR` with named constants to improve code readability and maintainability.